### DISCUSSION TITLE
Model Galleries

### DISCUSSION THREAD

**Discussion by WASasquatch** [civitai/civitai#234] [Ideas]
Allow users to post images to galleries for people to like/engage with. A way to show what a model can do without the obligatory review. Or perhaps user galleries that allow tagging a model, where a model page could have a tab for "_Images made with this model_" sort of deal.

**Comment by Machinekng** [comment]
Civitai made an announcement on reddit that this is a feature they have in development:

https://www.reddit.com/r/civitai/comments/10t0xdu/heres_how_were_planning_on_improving_browsability/
